File: src/main/kotlin/no/nav/arbeidsgiver/tiltakrefusjon/refusjon/SamtidigeEndringerException.java
package no.nav.arbeidsgiver.tiltakrefusjon.refusjon;

import no.nav.arbeidsgiver.tiltakrefusjon.Feilkode;
import no.nav.arbeidsgiver.tiltakrefusjon.FeilkodeException;

public class SamtidigeEndringerException extends FeilkodeException {

    public SamtidigeEndringerException() {
        super(Feilkode.SAMTIDIGE_ENDRINGER);
    }


}

File: src/main/kotlin/no/nav/arbeidsgiver/tiltakrefusjon/refusjon/SamtidigeEndringerException.kt
package no.nav.arbeidsgiver.tiltakrefusjon.refusjon

import no.nav.arbeidsgiver.tiltakrefusjon.Feilkode
import no.nav.arbeidsgiver.tiltakrefusjon.FeilkodeException

internal class SamtidigeEndringerException : FeilkodeException(Feilkode.SAMTIDIGE_ENDRINGER)